core/src/Pos/Aeson/Core/Configuration.hs

```diff
@@ -1,17 +1,86 @@
 {-# OPTIONS_GHC -fno-warn-orphans #-}
 
--- | Derived FromJSON instance for Pos.Core.Configuration.Configuration
-
 module Pos.Aeson.Core.Configuration
-    (
-    ) where
+     (
```

Comment:
The project standard seems to be to indent these in line with the module name. So I think 2 more spaces to the right.